uber/site_sections/accounts.py

```diff
@@ -201,6 +203,7 @@ def sitemap(self):
                 if getattr(method, 'exposed', False):
                     spec = inspect.getfullargspec(get_innermost(method))
                     if set(getattr(method, 'restricted', []) or []).intersection(AdminAccount.access_set()) \
+                            and not getattr(method, 'ajax', False) \
```

Comment:
Some of our `@ajax` methods were showing up on our site map, even though clicking the links would lead to an error. I couldn't find a good way to discern an `@ajax` method from a non-ajax method, so I just added an `ajax` flag.